internal/signer/darwin/keychain/keychain.go

```diff
@@ -405,3 +439,91 @@ func certIn(xc *x509.Certificate, xcs []*x509.Certificate) bool {
 	}
 	return false
 }
+
+func (k *Key) EncryptRSA(hashInput hash.Hash, random io.Reader, msg []byte) ([]byte, error) {
+	pub := k.Public()
+	rsaPubKey := pub.(*rsa.PublicKey)
+	return rsa.EncryptOAEP(hashInput, random, rsaPubKey, msg, nil)
+}
+
+func (k *Key) getRSAEncryptAlgor() (C.SecKeyAlgorithm, error) {
```

Comment:
nit: `Algor` is an uncommon abbreviation. My preference is to just write out `Algorithm`